src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -212,13 +212,14 @@ private void handleShowHelpEvent(ShowHelpRequestEvent event) {
     private void handlePersonPanelSelectionChangedEvent(PersonPanelSelectionChangedEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
         logic.updateFilteredTransactionList(event.getNewSelection().person);
-        logic.updateFilteredPersonList(event.getNewSelection().person);
+        logic.updateDebtorsList(event.getNewSelection().person);
+        logic.updateCreditorsList(event.getNewSelection().person);
     }
 
     @Subscribe
     private void handlePersonPanelNoSelectionEvent(PersonPanelNoSelectionEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
         logic.updateFilteredTransactionList();
-        logic.updateFilteredPersonList();
+        logic.updateDebtorsList();
     }
```

Comment:
you forgot : logic.updateCreditorsList();

Reply:
oh yes, amended!